ci-automation/ci-config.env

```diff
@@ -45,3 +45,9 @@ QEMU_BIOS="/usr/share/qemu/bios-256k.bin"
 # Published by vms.sh as part of the qemu vendor build.
 QEMU_UEFI_BIOS="flatcar_production_qemu_uefi_efi_code.fd"
 
+
+# -- Equinix Metal --
+
+EQUINIXMETAL_PARALLEL="${QEMU_PARALLEL}"
```

Comment:
For Azure, I've been using `QEMU_PARALLEL` too. Wondering if we should move out into a new var and use it?!

Reply:
Actually, `PARALLEL_TESTS` would be better (with a sane default if that variable is not set or empty), just like the qemu test does it: https://github.com/flatcar-linux/scripts/blob/main/ci-automation/ci-config.env#L47

Reply:
(e.g. `EQUINIXMETAL_PARALLEL="${PARALLEL_TESTS:-20}"` if "20" is a good parallel option for EM)